edscrapers/transformers/datajson/transform.py

```diff
@@ -51,8 +60,18 @@ def _transform_scraped_dataset(data, target_dept):
 
     dataset = Dataset()
 
-    dataset.landingPage = data.get('source_url')
-    dataset.title = data.get('title')
+    source_url = data.get('source_url')
+    if '|' in source_url:
```

Comment:
Can you explain this?

Reply:
Found some source URLs with a pipe '|' and the harvest couldn't validate these URLs.

Example:
https://ies.ed.gov/whatsnew/calendar/?id=2531&tid=0&cid=2&ts=6-2015-24|m&va=0

Changed to:
https://ies.ed.gov/whatsnew/calendar/?id=2531&tid=0&cid=2&ts=6-2015-24